### DISCUSSION TITLE
On Bootstrap (5) the lightbox a heref scroll auto to top on Page

### DISCUSSION THREAD

**Discussion by chrissen-77** [biati-digital/glightbox#377] [Q&A]
Hello,

i got a undesirable side-effect, on bootstrap (5). On click of one of the links, the page scoll auto to the top. now the users must allredy scroll down, afte ropen on of the galery.

any ideas/hints?

**Comment by gingerchew** [comment]
This sounds like it could just be a conflict with Bootstrap. Can you create an isolated code sample,like in a codepen for example?
